modules_core/org.openbravo.client.kernel/src/org/openbravo/client/kernel/ApplicationComponent.java

```diff
@@ -141,6 +142,17 @@ public String getButlerUtilsUrl() {
     return Utility.BUTLER_UTILS_URL;
   }
 
+  public String getLogoutRedirect() {
+    String logoutRedirectUrl;
+    try {
+      logoutRedirectUrl = Preferences.getPreferenceValue("LogoutRedirectURL", true, (Client) null,
```

Comment:
Unnecessary class cast in parameter (Client) null.

Reply:
Remember add a default value of this preference

Reply:
@Matias-Bernal If I don't cast the third parameter to 'Client', the IDE raises an error asking for the cast